snap/README.md

```diff
@@ -40,6 +40,24 @@ The App Service Configurable application service allows a variety of use cases t
 
 /var/snap/edgex-app-service-configurable/current/config/res/
 
+### Startup environment variables
+
+The startup behaviour can be modified through environment variables.
+
+To change the default startup duration (30 seconds), for a service to complete the startup, aka bootstrap, phase of execution by using the following command:
```

Comment:
The default in go-mod-bootstrap has been changed to 60 seconds.
https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/bootstrap/environment/variables.go#L34

Reply:
@lenny-intel Will update the value to 60. I got the 30 seconds value from https://docs.edgexfoundry.org/1.2/microservices/application/AdvancedTopics/

Reply:
I will add an issue to update it for Hanoi.